### ISSUE TITLE
`disableDarkMode` and `enableDarkMode` don't work on iOS 18 Simulator.

### ISSUE THREAD

**Issue by piotruela** [leancodepl/patrol#2393]
I recently ran the `service_dark_mode_test.dart` test from `e2e_app` project in the repository on the iOS 18 Simulator. The test fails at the stage where it attempts to click on the 'Developer' option in the settings.
<img width="670" alt="image" src="https://github.com/user-attachments/assets/1ce66002-9fef-4960-9d09-3dfd0ef44bf8">

Probably something has changed in how this element is described in iOS 18.

### Patrol version

patrol: 3.12.0
patrol_cli: 3.3.0

**Comment by Friesenjung** [comment]
+1 this is really annoying. Currently I cannot run our Patrol Test Suite due to this issue.

**Comment by Crawl3rGer** [comment]
Any updates on this one, @Kendru98 ? Facing the same issue. Thank you!

**Comment by Kendru98** [comment]
Already got the cause of issue, Developer page that we need to open is at the end of the list now, we need to add scroll there. Should be quick, but debugging which iOS version is differentcan be time-consuming 😓

**Comment by Kendru98** [comment]
Are you saying guys that this is broken since iOS 18?

**Comment by Crawl3rGer** [comment]
@Kendru98 yes, this is broken since iOS 18. We have used iOS 17 before and our system settings test worked fine.

**Comment by Kendru98** [comment]
Fix on the way, guys. Thanks for your patience. 😓

**Comment by Friesenjung** [comment]
How is it going? We are really looking forward to the fix :) ❤️

**Comment by Kendru98** [comment]
This week for sure, maybe even today 👍

**Comment by Kendru98** [comment]
@Friesenjung New patrol version was released today, please check if this fix/workaround working for you.

**Comment by Friesenjung** [comment]
@Kendru98 i will Check it later this day. Thanks so far!
